digdag-spi/src/main/java/io/digdag/spi/CommandExecutorRequest.java

```diff
@@ -0,0 +1,35 @@
+package io.digdag.spi;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface CommandExecutorRequest
+{
+    /**
+     * Returns a relative path of working dir from CommandExecutorContent.getLocalProjectPath().
+     *
+     * @return
+     */
+    Path getWorkingDirectory();
+
+    Map<String, String> getEnvironments();
+
+    List<String> getCommand();
```

Comment:
Should this be `getCommandLine()`?

Reply:
Thank you. that name is good. https://github.com/treasure-data/digdag/pull/835/commits/6c87e847e92358e9d26804d8d1543d87d2ba663e